vnext/local-cli/generator-windows/templates/cpp/src/AutolinkedNativeModules.g.h

```diff
@@ -1,17 +1,14 @@
-// AutolinkedNativeModules.g.h -- contents generated by "react-native run-windows"
-
-#include "pch.h"
-
+// AutolinkedNativeModules.g.h contents generated by "react-native autolink-windows"
 #pragma once
 
-// clang-format off
+#include "AutolinkedNativeModules.g.h"
 
+// clang-format off
 namespace winrt::Microsoft::ReactNative
 {
 
 static void RegisterAutolinkedNativeModulePackages(winrt::Windows::Foundation::Collections::IVector<winrt::Microsoft::ReactNative::IReactPackageProvider> const& packageProviders)
```

Comment:
this is providing an implementation but the .g.cpp file has the right one?

Reply:
Whoops.